lib/Serverless.js

```diff
@@ -64,6 +63,22 @@ class Serverless {
   }
 
   /**
+   * fill in the credentials by profile or by given credentials
+   */
+  _setCredentials() {
+      // Set Admin API Keys
+      var profiles = awsMisc.profilesMap();
+
+      if (typeof(process.env.SERVERLESS_ADMIN_AWS_PROFILE) != 'undefined' && typeof(profiles[process.env.SERVERLESS_ADMIN_AWS_PROFILE]) != 'undefined') {
```

Comment:
I'd just change this to:

```
if (!!process.env.SERVERLESS_ADMIN_AWS_PROFILE && !!profiles[process.env.SERVERLESS_ADMIN_AWS_PROFILE]) {
```

This will ensure it is defined and not empty